### PR TITLE
Feature/kitosudv 2114

### PR DESCRIPTION
**Pull request by mrjsawdk**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Core.DomainServices/GDPR/IDataProcessingRegistrationOversightDateAssignmentService.cs [hunk 1]
```diff
@@ -9,5 +9,6 @@ public interface IDataProcessingRegistrationOversightDateAssignmentService
         public Result<DataProcessingRegistrationOversightDate, OperationError> Assign(DataProcessingRegistration registration, DateTime oversightDate, string oversightRemark);
         public Result<DataProcessingRegistrationOversightDate, OperationError> Remove(DataProcessingRegistration registration, int oversightId);
         public Result<DataProcessingRegistrationOversightDate, OperationError> Modify(DataProcessingRegistration registration, int oversightId, DateTime oversightDate, string oversightRemark);
+        void Remove(DataProcessingRegistrationOversightDate oversight);
```
Comment (mrjsawdk) [inline Core.DomainServices/GDPR/IDataProcessingRegistrationOversightDateAssignmentService.cs:12]: Det der ser forkert ud... der er en "Remove ovenfor" som har "Result" så det lader til at der bliver blandet noget andet ind her end det der hører hjemme

File: Presentation.Web/Tests/02_Sequential/Local-admin/LocalAdmin.DataProcessing.e2e.spec.ts [hunk 2]
```diff
@@ -15,9 +15,9 @@ describe("Local admin is able to toggle DataProcessing", () => {
     var naviHelper = new NavigationBarHelper();
     var systemName = createItSystemName(1);
 
-    beforeAll(() => {
+    beforeAll(async () => {
```
Comment (mrjsawdk) [inline Presentation.Web/Tests/02_Sequential/Local-admin/LocalAdmin.DataProcessing.e2e.spec.ts:18]: async gør at man kan kalde await og så skal promise ikke returnes men awaites.

File: Tests.Unit.Presentation.Web/DomainServices/DPR/QQueryByAgreementConcludedTest.cs [hunk 3]
```diff
@@ -0,0 +1,59 @@
+﻿using Core.DomainModel.GDPR;
+using Core.DomainServices.Queries.DPR;
+using System.Linq;
+using Core.DomainModel.Shared;
+using Tests.Toolkit.Patterns;
+using Xunit;
+
+namespace Tests.Unit.Presentation.Web.DomainServices.DPR
+{
+    public class QQueryByAgreementConcludedTest : WithAutoFixture
```
Comment (mrjsawdk) [inline Tests.Unit.Presentation.Web/DomainServices/DPR/QQueryByAgreementConcludedTest.cs:10]: QQ

**Review by mrjsawdk — COMMENTED** [review]
småobservationer på den store branch

File: Presentation.Web/Tests/02_Sequential/Local-admin/LocalAdmin.DataProcessing.e2e.spec.ts (hunk 2 above)
Comment (mrjsawdk) [reply Presentation.Web/Tests/02_Sequential/Local-admin/LocalAdmin.DataProcessing.e2e.spec.ts:18]: mja den går åbenbart ikke uden lib "es2015" og så virker resten af kitos ikke
Til gengæld tager den en "done" så man kan få samme opførsel som async
image

**Review by JacobMalling — APPROVED** [review]

**Pull request merged**